CI/bench/timeseries/amlb_user_dir/constraints.yaml

```diff
@@ -0,0 +1,7 @@
+---
+
+test101:
```

Comment:
Yes, we need the constraint as this will be picked up by Bench running in our CI.
Basically the flow is as follows: we use our own custom configs to specify constraints, presets, frameworks and benchmark datasets, and for bench to pick this up we need to store all this in something called _amlb-user-dir_, then a cloud config file is generated which ensures the custom configs are picked up rather than the default AMLB configs.
If nothing is provided, AMLB Configs are picked up by default.

Reply:
I see, so [these configs](https://github.com/autogluon/autogluon-bench/blob/master/sample_configs/amlb_configs/constraints.yaml#L3) are available in the `autogluon.bench` repo, but they aren't automatically downloaded when we install bench with `pip install autogluon.bench`. So we need to store these files somewhere to use them, right?